packages/amplication-git-pull-request-service/src/pull-request/pull-request.service.ts

```diff
@@ -46,7 +46,7 @@ export class PullRequestService {
     const gitClientService = await new GitClientService().create(
       {
         provider: gitProvider,
-        installationId,
+        providerProperties: { installationId },
```

Comment:
For now, this is OK, but we need to think about how to use the git-util from services other than the amplication-server. where do we get the providerProperties from etc

Reply:
git-pull-request is also using it

Reply:
I meant to think about how we get the properties from the DB and pass them to git-util